packages/squidex/test/upsert.test.ts

```diff
@@ -47,6 +50,66 @@ describe('squidex wrapper - upsert', () => {
     ).rejects.toThrow('Bad Request');
   });
 
+  it('returns 400 along with the response payload formatted to json when squidex returns validation error', async () => {
+    nock(config.baseUrl)
+      .patch(`/api/content/${config.appName}/${collection}/${contentId}`)
+      .query(() => true)
+      .reply(400, {
+        message: 'Validation error',
+        traceId: '00-ba8100d975b2cb551a023702a7d0d5b7-891e647127349001-01',
+        type: 'https://tools.ietf.org/html/rfc7231#section-6.5.1',
+        details: ['link.iv: Another content with the same value exists.'],
+        statusCode: 400,
+      });
+
+    await expect(() =>
+      client.upsert(
+        contentId,
+        JSON.parse(
```

Comment:
what is the purpose of this JSON.parse(JSON.stringify(x))?

Reply:
you should probably ask the committers of the rest of these tests :D normally this would be to avoid mutation but since we are actually creating the object inline i dont really know what its doing here :D